ipie/utils/tests/test_io.py

```diff
@@ -23,6 +23,39 @@
 
 from ipie.utils.io import read_hamiltonian, read_wavefunction, write_hamiltonian, write_wavefunction
 from ipie.utils.testing import get_random_phmsd_opt
+import h5py
+from ipie.utils.chunk_large_chol import split_cholesky
+
+
+@pytest.mark.unit
+def test_split_cholesky():
+    naux = 105
+    nbas = 10
+    temp_hdf5_filename = "temp_chol_test.h5"
+    mock_data = np.random.rand(naux, nbas, nbas)
+
+    with h5py.File(temp_hdf5_filename, "w") as f:
```

Comment:
Use a NamedTemporaryFile similar to the other tests to get a tmp file without needing to manually delete things yourself.

Reply:
thanks! 
just modified.